drivers/spi/spi_nxp_lpspi/spi_nxp_lpspi_common.c

```diff
@@ -53,6 +53,34 @@ static inline clock_ip_name_t lpspi_get_clock(LPSPI_Type *const base)
 }
 #endif
 
+/* Issue a TCR is requirement on some version of LPSPI or last RX DMA transfer will
+ * never end. It also depends on flag SPI_HOLD_ON_CS, if it is set, we will try to
+ * keep PCS asserted during transanctions.
+ * On RT platform, main version is 01: TCR is needed.
```

Comment:
RT is ambiguous for example RT700 does has version 02.
Could you be more specific about the product families and also include lines such S32K and Kinetis.

Reply:
Good point. I have not have chance to have a look at all main SoC's LPSPI version. Will give more details.

Reply:
Done, please review again